src/main/java/command/DeadlineCommand.java

```diff
@@ -0,0 +1,34 @@
+package command;
+
+import java.time.LocalDateTime;
+
+import mike.TaskList;
+import task.DeadlineTask;
+
+public class DeadlineCommand extends AddCommand {
```

Comment:
Apparently it's discouraged to use public class unless it's a class that's purely for data 

https://se-education.org/guides/conventions/java/intermediate.html#variables